example/hybrid-mbr.nix

```diff
@@ -0,0 +1,62 @@
+{ disks ? [ "/dev/sda" ], ... }:
+{
+  disk = {
+    main = {
+      type = "disk";
+      device = builtins.elemAt disks 0;
+      content = {
+        type = "hybrid_table";
+        efiGptPartitionFirst = false;
+        hybrid_partitions = [
+          {
+            type = "hybrid_partition";
+            gptPartitionNumber = 1;
```

Comment:
isn't the index in the list enough? why do we need to count here? what would happen if I set this to 3?

Reply:
You need to choose which GPT partition want to make available in DOS table. Maybe a sane default using index is ok.

Reply:
correct.